tools/ou_rs_amplitude_retune_sweep.py

```
@@ -0,0 +1,234 @@
+FINE_SIGMA = (0.22, 0.27, 0.32, 0.38, 0.45, 0.64, 0.90, 1.80)
+FINE_R = (12.1, 14.4, 17.11, 20.3, 24.2)
+
+
+def run_record(binary: Path, record: Path, env_extra: dict[str, str]) -> dict[str, float]:
+    env = dict(os.environ)
+    env["W3D_WRITE_TIMESERIES"] = "0"
+    # Score every record, including any that trips a historical gate: the
+    # sweep reports what the filter did, not whether it passed.
+    env["W3D_COLLECT_ALL_GATES"] = "1"
+    env.update(env_extra)
+    completed = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)